api.md

```diff
@@ -81,15 +85,23 @@
 
 **Response:**
 
+
 - **200:**
 
 ```json
 {
-  "msg": "Image updated successfully !",
```

Comment:
Why did u remove this ?

Reply:
200 means it OK its done, i think no need for this extra line.

Reply:
This will be shown as it's

Reply:
The front will show the `msg`  not a text as it's

Reply:
+ i already sent you the new image , it's obvious really!

Reply:
ok